README.md

```diff
@@ -14,7 +14,10 @@ Update the configuration file under `./server/config.json`:
 
 ```json
 {
-  "AUTHORIZE_API_KEY": "mysecret",
+  "AUTHZ_API_CLIENT_ID": "optional. client id of authz api client",
+  "AUTHZ_API_CLIENT_SECRET": "optional. client secret of authz api client",
+  "AUTHZ_API_AUDIENCE": "optional. urn:auth0-authz-api",
```

Comment:
an additional setting is not required. this should just use `urn:auth0-authz-api`.